sockeye/coverage.py

```diff
@@ -53,14 +56,16 @@ def get_coverage(config: CoverageConfig) -> 'Coverage':
     :param config: Coverage configuration.
     :return: Instance of Coverage.
     """
-    if config.type == 'count':
-        utils.check_condition(config.num_hidden == 1, "Count coverage requires coverage_num_hidden==1")
+    if config.type == 'count' or config.type == 'fertility':
```

Comment:
if you are at it, it'd be great if you could change to string comparisons to use the newly introduced constants defined in constants.py